R/forecast-bbs-core.R

```diff
@@ -113,15 +119,17 @@ get_bbs_data <- function(start_yr, end_yr, min_num_yrs){
                          JOIN bbs.routes
                            ON bbs.counts.statenum=bbs.routes.statenum
                            AND bbs.counts.route=bbs.routes.route
-                         WHERE bbs.weather.runtype=1 AND bbs.weather.rpid=101;"
+                         WHERE bbs.weather.runtype=1 AND bbs.weather.rpid=101
+                           AND bbs.weather.statenum!=3 AND bbs.weather.countrynum=840;"
```

Comment:
Do we want these filters to be explicit or do we just want the lack of available environmental data to take care of them? My first instinct is to let the availability of environmental data take care of the filtering so that if we go back and at point and add environmental data outside of the contiguous US we automatically start folding that ecological data in as well.